### PR TITLE
Refactor/sponsor style

### PR DESCRIPTION
**Pull request by baby230211**
## Types of changes

- [ ] **Bugfix**
- [ ] **New feature**
- [x] **Refactoring**
- [ ] **Breaking change** (any change that would cause existing functionality to not work as expected)
- [ ] **Documentation Update**
- [ ] **Other (please describe)**

## Description

1. fix sponsor card hover transition
2. add prettier tailiwind plugin to automate sort the class [ref](https://tailwindcss.com/blog/automatic-class-sorting-with-prettier)

## Expected behavior

x | Mobile | Desktop
-- | -- | --
ScreenShot | 截圖 2023-05-06 下午3 53 20 | 截圖 2023-05-06 下午3 53 30

### CODE REVIEW

File: pages/index.vue [hunk 1]
```diff
@@ -79,7 +62,7 @@
             <div
                 v-if="showIndexSponsorSection"
                 id="sponsor"
-                class="pt-12 lg:mx-auto lg:w-full;"
+                class="lg:w-full; pt-12 lg:mx-auto"
```
Comment (yungshenglu) [inline pages/index.vue:65]: Should we need `;` ?

**Comment by yungshenglu** [comment]
盡量不要再用 class 名稱包 Tailwind Style
如果有看到以前這樣的寫法，可以幫忙改回去，直接寫在 class 就好

File: pages/index.vue (hunk 1 above)
Comment (baby230211) [reply pages/index.vue:65]: this is tailwind linter cli changed.

**Comment by baby230211** [comment]
> 盡量不要再用 class 名稱包 Tailwind Style 如果有看到以前這樣的寫法，可以幫忙改回去，直接寫在 class 就好

the change is updated on `SponsorCardCollection.vue`

**Review by yungshenglu — APPROVED** [review]

**Pull request merged**